spec/features/equipment_model_views_spec.rb

```diff
@@ -3,9 +3,68 @@
 describe 'Equipment model views' do
   subject { page }
 
+  shared_examples 'can view detailed table' do
+    it { should have_selector 'th', text: 'Available' }
+  end
+
+  shared_examples 'can view full and detailed andtable' do
+    it_behaves_like 'can view detailed table'
+    it { expect(page).to have_link 'Edit' }
+  end
+
   context 'index view' do
     before { visit equipment_models_path }
     it { is_expected.to have_content('Equipment Models') }
     it { is_expected.to have_content(@eq_model.name) }
   end
+  # Testing for issue 1199
+  context 'check for super user' do
+    before do
+      sign_in_as_user(@superuser)
+      visit equipment_models_path
+    end
+    after { sign_out }
+    it_behaves_like 'can view full and detailed andtable'
+  end
+
+  context 'check for admin' do
+    before do
+      sign_in_as_user(@admin)
+      visit equipment_models_path
+    end
+    after { sign_out }
+    it_behaves_like 'can view full and detailed andtable'
+  end
+
+  context 'check for patron' do
+    before do
+      sign_in_as_user(@user)
+      visit equipment_models_path
+    end
+    after { sign_out }
+    it_behaves_like 'can view detailed table'
+  end
+
+  context 'check for check out person' do
+    before do
+      sign_in_as_user(@checkout_person)
+      visit equipment_models_path
+    end
+    after { sign_out }
+    it_behaves_like 'can view detailed table'
+  end
+
+  context 'check for guest' do
+    it { should_not have_selector 'th', text: 'Available' }
+  end
+
+  context 'check for banned user' do
+    before do
+      @banned = FactoryGirl.create(:banned)
```

Comment:
Let's move this to `spec/support/feature_helpers.rb` in the `app_setup` method - I can imagine we might want to use banned users in other tests as well so no reason not to do it each time.